src/panels/lovelace/badges/hui-summary-badge.ts

```diff
@@ -9,16 +9,21 @@ import { computeDomain } from "../../../common/entity/compute_domain";
 import "../../../components/ha-label-badge";
 import { HomeAssistant } from "../../../types";
 import { LovelaceBadge } from "../types";
-import { LightsBadgeConfig } from "./types";
+import { SummaryBadgeConfig } from "./types";
 
-@customElement("hui-lights-badge")
-export class HuiLightsBadge extends LitElement implements LovelaceBadge {
+@customElement("hui-summary-badge")
+export class HuiSummaryBadge extends LitElement implements LovelaceBadge {
   @property({ attribute: false }) public hass?: HomeAssistant;
 
-  @property() protected _config?: LightsBadgeConfig;
+  @property() protected _config?: SummaryBadgeConfig;
 
-  public setConfig(config: LightsBadgeConfig): void {
-    this._config = { icon: "hass:lightbulb-group", ...config };
+  public setConfig(config: SummaryBadgeConfig): void {
+    this._config = {
+      icon: "hass:lightbulb-group",
+      domain: "light",
+      state: "on",
```

Comment:
should this be default? Or should the default come from the domain?

Reply:
Talking about the icon

Reply:
:shrug: Like add a map for "on" states for all possible domains?

Reply:
No sorry I Fucked the comment lines up. I mean the Icon only

Reply:
```suggestion
      icon: "hass:lightbulb-group",
      domain: "light",
      state: computeOnState(config.domain),
```
?